src/js/modules/infragistics.ui.scroll.js

```diff
@@ -1197,7 +1197,7 @@
 		_linkElementsH: function (inElements) {
 			this._linkedHElems = [];
 			if (inElements) {
-				for (var index in inElements) {
+				for (var index = 0; index < inElements.length; index++) {
```

Comment:
Using the normal "for" loop also fixes the issue.
The "for..in" loop is usually used when going through properties of an object. In the cases I've used it normally it is expected to be an array so I think it more proper to use the normal for loop.